Refactor: Replace Fixtures with static values to Constants

Daksh2000 opened the pull request:
Issue - https://github.com/pypi/warehouse/issues/16913
In this PR, I have created a new constants.py file under tests directory and declared all the constants there and then replaced the references of fixtures with these constants.

Daksh2000 commented:
> Issue - #16913 In this PR, I have created a new constants.py file under tests directory and declared all the constants there and then replaced the references of fixtures with these constants.

Hi everyone, please have a look at this PR, and could someone please approve the pending workflows

miketheman commented on tests/common/constants.py:
```diff
@@ -0,0 +1,51 @@
+
+REMOTE_ADDR = "1.2.3.4"
+
+REMOTE_ADDR_HASHED = "6694f83c9f476da31f5df6bcc520034e7e57d421d247b9d34f49edbfc84a764c"
```
Add comments above constants that explain the value, like this one (update for accuracy):
https://github.com/pypi/warehouse/blob/834086fef45010864aca9c345dd8f3e30bc60665/tests/conftest.py#L130-L131

Daksh2000 replied:
Done

miketheman reviewed (commented):
Hey @Daksh2000 !

This is a good start, but I don't see any cases where you've removed the original fixture from `conftest.py` to be replaced by the constants. Is that change left out intentionally?

Daksh2000 commented:
> Hey @Daksh2000 !
> 
> This is a good start, but I don't see any cases where you've removed the original fixture from `conftest.py` to be replaced by the constants. Is that change left out intentionally?

Yes , I did leave it intentionally in case it would be deprecated later on, okay I will completely remove those fixtures.

Daksh2000 commented:
> > Hey @Daksh2000 !
> > This is a good start, but I don't see any cases where you've removed the original fixture from `conftest.py` to be replaced by the constants. Is that change left out intentionally?
> 
> Yes , I did leave it intentionally in case it would be deprecated later on, okay I will completely remove those fixtures.



> Hey @Daksh2000 !
> 
> This is a good start, but I don't see any cases where you've removed the original fixture from `conftest.py` to be replaced by the constants. Is that change left out intentionally?

Hi @miketheman , 
I have removed the fixtures and changed test cases accordingly which were using the fixtures and added comments as well to the constants,
Please share your views

miketheman reviewed (changes requested):
Nice work! I left a couple of suggestions on updates to the comment for accuracy, so someone could actually run those commands to produce the same result.

There's also some tests in `tests/unit/ip_addresses/test_models.py` that currently use a static string of `"1.2.3.4"` - had you thought about replacing those as well with `REMOTE_ADDR`?

With these changes, I think you're almost there!

Daksh2000 commented:
> Nice work! I left a couple of suggestions on updates to the comment for accuracy, so someone could actually run those commands to produce the same result.
> 
> There's also some tests in `tests/unit/ip_addresses/test_models.py` that currently use a static string of `"1.2.3.4"` - had you thought about replacing those as well with `REMOTE_ADDR`?
> 
> With these changes, I think you're almost there!

Thanks for pointing this out, I replaced all the instances of remote_addr, but forgot to lookup for the values as well like "1.2.3.4", I will make the desired changes.

Daksh2000 commented on tests/common/constants.py:
```diff
@@ -0,0 +1,109 @@
+REMOTE_ADDR_HASHED = "6694f83c9f476da31f5df6bcc520034e7e57d421d247b9d34f49edbfc84a764c"
+
+"""
+Output of `hashlib.sha256((remote_addr + "pepa").encode("utf8")).hexdigest()`
```
Done

Daksh2000 commented on tests/common/constants.py:
```diff
@@ -0,0 +1,109 @@
+REMOTE_ADDR = "1.2.3.4"
+
+"""
+Static output of `hashlib.sha256(remote_addr.encode("utf8")).hexdigest()`
```
Done

Daksh2000 commented:
Hi @miketheman , thank you for helping me contribute !!
